models/src/main/thrift/content/v1.thrift

```diff
@@ -898,7 +898,7 @@ struct ListItem {
 
     2: optional string title;
 
-    3: optional list<Tag> contributors;
+    3: optional list<string> contributorIds;
```

Comment:
Are we okay to drop the old field given there are no multi-byline articles in PROD?

Reply:
As I look at this, I do think it might be better to comment out 
`3: optional list<Tag> contributors;` 
and add 
`4: optional list<string> contributorIds;` 
in order to record the fact that it has changed, even though it was never used.

Let's think on it a bit before committing either way?

Reply:
Thanks very much for looking into this.

> `4: optional list<string> contributorIds;`

Did you intend to use the field ID `4` here? Can we reuse `3`? Otherwise perhaps it should be `9` to avoid clashes with other fields?

> Let's think on it a bit before committing either way?

For sure - I appreciate the need for caution here

Reply:
Hi again @JustinPinner, I've discussed this with @davidfurey and we agree with your approach above is needed, so I've made the change